src/style.css

```diff
@@ -151,15 +151,21 @@
 }
 
 .greenstand-point-selected-box {
```

Comment:
Sorry, it's my fault, I didn't emphasize that we need still to keep the old version code here, our plan is to be able to switch between these different versions (because our online version now is using this core lib too), so can you keep the old CSS class, and create a new one/classes for the new icon?

I opened a new issue for the next steps: https://github.com/Greenstand/treetracker-web-map-core/issues/61

Reply:
> Sorry, it's my fault, I didn't emphasize that we need still to keep the old version code here, our plan is to be able to switch between these different versions (because our online version now is using this core lib too), so can you keep the old CSS class, and create a new one/classes for the new icon?
> 
> I opened a new issue for the next steps: #61

No problem, i will keep the old ones

Reply:
Made the changes and kept the old CSS styles. The map will display the new icons for now. I will take #61 too and add an option to choose between different versions of icon